apps/app/components/welcome/featured/dreamshaper.tsx

```diff
@@ -877,30 +900,34 @@ export default function Dreamshaper({
 
         {!isMobile && <Separator orientation="vertical" className="h-6 mr-2" />}
 
-        <Tooltip>
+        <Tooltip delayDuration={50}>
           <TooltipTrigger asChild>
-            <Button
-              disabled={
-                updating || !inputValue || profanity || exceedsMaxLength
-              }
-              onClick={e => {
-                e.preventDefault();
-                submitPrompt();
-              }}
-              className={cn(
-                "border-none w-36 items-center justify-center font-semibold text-xs bg-[#00EB88] flex",
-                updating && "bg-muted text-muted-foreground",
-                isMobile ? "w-12 rounded-lg" : "w-12 rounded-lg",
-              )}
-            >
-              {updating ? (
-                <Loader2 className="h-4 w-4 animate-spin" />
-              ) : (
-                <Send className="h-4 w-4 stroke-[3]" />
-              )}
-            </Button>
+            <div className="relative inline-block">
+              <Button
+                disabled={
+                  updating || !inputValue || profanity || exceedsMaxLength
+                }
+                onClick={e => {
+                  e.preventDefault();
+                  submitPrompt();
+                }}
+                className={cn(
+                  "border-none w-36 items-center justify-center font-semibold text-xs bg-[#00EB88] flex",
+                  updating && "bg-muted text-muted-foreground",
+                  isMobile ? "w-12 rounded-lg" : "w-12 rounded-lg",
+                )}
+              >
+                {updating ? (
+                  <Loader2 className="h-4 w-4 animate-spin" />
+                ) : (
+                  <Send className="h-4 w-4 stroke-[3]" />
+                )}
+              </Button>
+            </div>
           </TooltipTrigger>
-          <TooltipContent>Press Enter</TooltipContent>
+          <TooltipContent side="top" className="bg-white text-black border border-gray-200 shadow-md dark:bg-zinc-900 dark:text-white dark:border-zinc-700">
+            Prompt <span className="text-gray-400 dark:text-gray-500">Enter</span>
```

Comment:
WDYT about this? I think "prompt" as a verb might be a little technical for new users
```suggestion
            Update <span className="text-gray-400 dark:text-gray-500">(Enter)</span>
```